### DISCUSSION TITLE
Don't do a preview deploy of draft pull requests

### DISCUSSION THREAD

**Discussion by mellson** [vercel/community#767] [Help] [answered]
Is it possible to stop Vercel from creating preview deploys of pull requests still in draft?

**Comment by amyegan** [comment, marked as answer]
You can use [Ignored Build Step](https://vercel.com/support/articles/how-do-i-use-the-ignored-build-step-field-on-vercel) to skip builds based on parameters you set. There are many ways it can be configured so you have a lot of flexibility with that approach. A few examples for reference:
- Only deploy production environment commits: [How do I use the "Ignored Build Step" field on Vercel?](https://vercel.com/support/articles/how-do-i-use-the-ignored-build-step-field-on-vercel)
- Using something like [skip deploy] or [draft] in PR title: https://github.com/vercel/community/discussions/60#discussioncomment-3221395
- Only deploy for specific branch names: https://github.com/vercel/vercel/issues/3166#issuecomment-834562483

You also have the option to [disable Vercel for GitHub](https://vercel.com/docs/project-configuration#git-configuration/github-enabled) automatic deployments and [customize your own GitHub Actions with Vercel](https://vercel.com/support/articles/how-can-i-use-github-actions-with-vercel) for even greater control over how and when your code is built and deployed.
